doc/source/whatsnew/v1.1.3.rst

```diff
@@ -73,4 +73,4 @@ Other
 Contributors
 ~~~~~~~~~~~~
 
-.. contributors:: v1.1.2..v1.1.3|HEAD
```

Comment:
Can you also remove this removal (and leave this for the actual release commits which also update the date etc), then it should pass?

Reply:
There's also a reference to v1.1.3 down below, should that also be removed, or do we wait for the tag?